tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessEnvironment.cs

```diff
@@ -33,15 +33,23 @@ public static string[] ReadModules(Process process)
         {
             if (RuntimeInformation.IsOSPlatform(System.Runtime.InteropServices.OSPlatform.Linux))
             {
-                // On Linux, Process.Modules does not list dynamically loaded assemblues: https://github.com/dotnet/runtime/issues/64042
+                // On Linux, Process.Modules does not list dynamically loaded assemblies: https://github.com/dotnet/runtime/issues/64042
```

Comment:
I got the assem-blues

![image](https://user-images.githubusercontent.com/18755388/154136093-830c68eb-3f11-4182-8b7f-505647d3b108.png)